### PR TITLE
Correct a bug on annotated field

### PR DESCRIPTION
**Pull request by Cafeinoman**
Found a bug that need to be corrected before release...

### CODE REVIEW

File: src/main/java/io/vertx/codegen/type/ClassTypeInfo.java [hunk 1]
```diff
@@ -97,6 +99,11 @@ public boolean equals(Object obj) {
   }
 
   @Override
+  public List<AnnotationValueInfo> getAnnotations() {
```
Comment (vietj) [inline src/main/java/io/vertx/codegen/type/ClassTypeInfo.java]: missing javadoc, which annotations are returned ?

File: src/main/java/io/vertx/codegen/type/TypeInfo.java [hunk 2]
```diff
@@ -52,6 +54,10 @@ public String getName() {
     return format(true);
   }
 
+  public List<AnnotationValueInfo> getAnnotations(){
```
Comment (vietj) [inline src/main/java/io/vertx/codegen/type/TypeInfo.java]: missing javadoc, which annotations are returned ?

**Comment by vietj** [comment]
it seems there are tests only for data object model and not the other models

**Comment by Cafeinoman** [comment]
Correction on #128

**Pull request closed without merging**